### DISCUSSION TITLE
Welcome to cvsTrainer Discussions!

### DISCUSSION THREAD

**Discussion by cvs0** [cvs0/cvsTrainer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jorgem3killer** [comment]
Hello friend, I have seen your GitHub project, the RDR Trainer in C++/C, without barely knowing how to program, I would like to modify it to make Teleports in another different game and similar tricks, the thing is that whenever I download an open source project to modify it and use it, I have errors, it doesn't compile, etc..

I have Windows 10, could you tell me exactly with what programming program, extensions and so on you have developed your Trainer so that I can investigate without errors as soon as I open the project? thank you!!

**Reply by cvs0** [reply]
Hey! Thanks for the message. I have been using Visual Studio 2022. Just checking, but do you mean you want to make it for another game that is not RDR2?

**Reply by jorgem3killer** [reply]
> Hey! Thanks for the message. I have been using Visual Studio 2022. Just checking, but do you mean you want to make it for another game that is not RDR2?

Exactly, I would like to learn with your trainer's base, to be able to do teleports and similar things in a different game than the one I usually play...

**Reply by cvs0** [reply]
> > Hey! Thanks for the message. I have been using Visual Studio 2022. Just checking, but do you mean you want to make it for another game that is not RDR2?
> 
> Exactly, I would like to learn with your trainer's base, to be able to do teleports and similar things in a different game than the one I usually play...

Well, you cant use this code for other games. All game engines work differently. This is specifically for rdr2. Nothing else.

**Comment by jorgem3killer** [comment]
Hello, I get 338 several errors when running the project in VS2022 at "script.cpp", how can I solve them bro?

Codes of error:
E1696
E0276
E0020
E0441
E0065
E0289
C1083

**Reply by cvs0** [reply]
Sorry, but I do not have the actual error logs. From this what it seems is that you haven't set up your ScriptHookRDR2 development environment. I can send you over the resources to get started.

**Reply by cvs0** [reply]
You need the ScriptHookRDR2 SDK from Alexander Blade.

Link: [http://dev-c.com/rdr2/scripthookrdr2/](http://dev-c.com/rdr2/scripthookrdr2/)
